shared/components/label/Label.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Label = styled.label`
+  color: #7f8fa4;
+  display: flex;
+  flex-direction: column;
+  font-size: 14px;
+  margin-bottom: 10px;
+  width: 100%;
+
+  @media only screen and (min-width: 901px) {
+    margin-bottom: 20px;
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+
+  @media only screen and (min-width: 901px) {
+    margin: ${props => props.halfWidth && '0'};
```

Comment:
What happens if `halfWidth` is `false`?

Reply:
When `halfWidth` is `false`, the label is full width on desktop.

Reply:
And what is the value of the property `width` then?